client/overview/style.scss

```diff
@@ -0,0 +1,10 @@
+// Gaps.
+$gap-largest: 40px;
+$gap-larger: 36px;
+$gap-large: 24px;
+$gap: 16px;
+$gap-small: 12px;
+$gap-smaller: 8px;
+$gap-smallest: 4px;
```

Comment:
Should we move the `$gap-*` variables to it's own `_variables.scss` under `stylesheets/abstracts`? Given we want to use these variables for other components as well.

Reply:
Moved in 6070701.